### ISSUE TITLE
Add exit rule assertion to strategy skills

### ISSUE THREAD

**Issue by nix378031** [nix378031/nix-ops#33]
---
model: anthropic/claude-sonnet-4-20250514
channel: "#nix-trades"
context_files:
  - trading/audit/paper-trading-audit-2026-02-13.md
  - trading/strategy.md
  - skills/nix-put-verticals/SKILL.md
  - skills/nix-calendars/SKILL.md
  - skills/nix-earnings/SKILL.md
---

## Objective
Add mandatory exit rule assertion to strategy skills. Before closing ANY position, the agent must explicitly name which exit rule applies AND verify the numerical condition is met.

## Background
Paper trading audit (Feb 13) found 2/4 trades closed without any documented exit rule being triggered. The SPY vertical was labeled PROFIT_TARGET while the position was losing money. The GLD calendar was labeled MANUAL_CLOSE with no rule justification. Agents are closing positions on vibes, not rules.

### Exit Rule Assertion Block
Add to each strategy skill, BEFORE any close order submission:

```
### Exit Rule Assertion (MANDATORY — before ANY close)
Before submitting a close order, you MUST:

1. **Name the rule:** State which specific exit rule from strategy.md applies
2. **Show the math:** Calculate the exact condition value
3. **Verify the threshold:** Confirm the condition is met

Example (put vertical):
- Rule: 50% profit target
- Math: Current spread value $0.35 vs entry credit $0.78 → profit = $0.43 = 55.1% of credit
- Threshold: 55.1% > 50% ✅ → CLOSE

Example (calendar):
- Rule: 25-50% profit target  
- Math: Current spread value $5.80 vs entry debit $4.30 → profit = $1.50 = 34.9% of debit
- Threshold: 34.9% is in 25-50% range ✅ → CLOSE

If NO exit rule condition is met, DO NOT CLOSE. Log the position status and move on.
```

### Strategy-Specific Exit Rules (from strategy.md)

**Put Verticals:**
- 50% of credit received (profit target)
- 2x credit received loss (loss stop)  
- 21 DTE remaining (time stop)

**Calendars:**
- 25-50% of debit paid (profit target)
- Front-month expiration (time stop)
- Max loss = debit paid

**Earnings (strangle/IB):**
- Close day after earnings (time-based)
- ≤1 DTE (time stop)

**VIX Hedge:**
- Hold to expiry (default)
- VIX > 40 + spread > 80% max value (early close)

### What NOT to change
- Do not change the exit rules themselves
- Do not change order execution logic
- Only add the assertion checkpoint BEFORE the close decision

## Acceptance Criteria
- [ ] All 3 active strategy skills updated (put-verticals, calendars, earnings)
- [ ] VIX hedge skill updated (simpler — mostly hold to expiry)
- [ ] Assertion block includes: rule name, numerical calculation, threshold check
- [ ] If no rule is met, agent logs status but does NOT close
- [ ] trades.jsonl exit entries must include `exit_rule_assertion` field showing the math
- [ ] Committed and pushed
